### PR TITLE
Administrator password change

### PR DESCRIPTION
**Pull request by The10FpsGuy**
i havent tested it and i suck at making readme.md files.... sorry

### CODE REVIEW

**Comment by The10FpsGuy** [comment]
i reeeeaaally want to thank squibs

File: payloads/library/credentials/Payload 2.txt [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+#activate stickykeys
+Q SHIFT
+Q SHIFT
+Q SHIFT
+Q SHIFT
+Q SHIFT
+Q SHIFT
+Q SHIFT
+Q SHIFT
+Q DELEY 100
```
Comment (Layer8Err) [inline payloads/library/credentials/Payload 2.txt]: DELAY

Comment (The10FpsGuy) [inline payloads/library/credentials/Payload 2.txt]: i suck at writing deley.......

Comment (sebkinne) [inline payloads/library/credentials/Payload 2.txt]: @The10FpsGuy once this is fixed we'll accept the PR.

**Comment by sebkinne** [comment]
Hi @The10FpsGuy there are a couple of issues with your payload:

1. It is not in it's own folder. Please ensure that your payload is placed in a folder with the payload's name.
2. There is a typo in your payload (DELEY vs DELAY).

Once these issues are fixed, we'll review it again.

**Pull request closed without merging**